btcjson/walletsvrcmds_test.go

```diff
@@ -1153,6 +1154,17 @@ func TestWalletSvrCmds(t *testing.T) {
 			},
 		},
 		{
+			name: "version",
```

Comment:
Same thing here.  This should all go in `btcdextcmds_test.go` since it is a btcd-specific extension.